amp.js

```diff
@@ -49,6 +49,7 @@ createTask('check-owners', 'checkOwners');
 createTask('check-renovate-config', 'checkRenovateConfig');
 createTask('check-sourcemaps', 'checkSourcemaps');
 createTask('check-types', 'checkTypes');
+createTask('check-update-design-review-issues', 'check');
```

Comment:
It seems weird that the actual script that creates these issues is not an `amp` task, and we have a separate task to run its tests.

I propose that the issue creation be refactored into an `amp` task called `create-design-review-issues`, and the tests be rolled into the same task under a `--test` flag, similar to `amp make-extension --test`.